<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#1E1E1E;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="heading_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:20px;
font-weight:bold;
qproperty-alignment: 'AlignCenter';
margin-bottom:15px;</string>
      </property>
      <property name="text">
       <string>System Monitor App</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,3,3">
        <property name="leftMargin">
         <number>60</number>
        </property>
        <property name="rightMargin">
         <number>45</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0">
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">
font-size:12px;
font-weight:bold;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="styleSheet">
             <string notr="true">
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="styleSheet">
                <string notr="true">color:white;
font-size:12px;
font-weight:bold;
 qproperty-alignment: 'AlignCenter';</string>
               </property>
               <property name="text">
                <string>Used RAM (MB)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="RAM_label">
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">color : red;
font-size:20px;
font-weight:bold;

 qproperty-alignment: 'AlignCenter';</string>
               </property>
               <property name="text">
                <string>loading...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="styleSheet">
             <string notr="true">/*
background-color: #100638;
background-image: linear-gradient(170deg, #100638 0%, #073631 100%);
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="styleSheet">
                <string notr="true">color:white;
font-size:12px;
font-weight:bold;
 qproperty-alignment: 'AlignCenter';</string>
               </property>
               <property name="text">
                <string>Process Count</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="Process_label">
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">color : red;
font-size:20px;
font-weight:bold;

 qproperty-alignment: 'AlignCenter';</string>
               </property>
               <property name="text">
                <string>loading...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="styleSheet">
             <string notr="true">/*
background-color: #100638;
background-image: linear-gradient(170deg, #100638 0%, #073631 100%);
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>7</number>
             </property>
             <property name="bottomMargin">
              <number>11</number>
             </property>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="styleSheet">
                <string notr="true">color:white
;

font-size:12px;
font-weight:bold;
 qproperty-alignment: 'AlignCenter';</string>
               </property>
               <property name="text">
                <string>Thread Count</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="Thread_label">
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">color : red;
font-size:20px;
font-weight:bold;
 qproperty-alignment: 'AlignCenter';</string>
               </property>
               <property name="text">
                <string>loading...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <property name="styleSheet">
             <string notr="true">/*
background-color: #100638;
background-image: linear-gradient(170deg, #100638 0%, #073631 100%);
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="styleSheet">
                <string notr="true">color:white;

font-size:12px;
font-weight:bold;
 qproperty-alignment: 'AlignCenter';</string>
               </property>
               <property name="text">
                <string>Handle Count</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="Handles_label">
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">color : red;
font-size:20px;
font-weight:bold;
 qproperty-alignment: 'AlignCenter';</string>
               </property>
               <property name="text">
                <string>loading...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>25</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5">
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">
font-size:12px;
font-weight:bold;</string>
          </property>
          <property name="text">
           <string>CPU Usage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="cpuProgressBar">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,5">
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">
font-size:12px;
font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Memory </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="memoryProgressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,5">
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">
font-size:12px;
font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Disk</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="diskProgressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
